src/client/src/apps/SpotlightRoute/intent-handlers/showBlotter.ts

```diff
@@ -1,27 +1,50 @@
-import { InteropTopics, PlatformAdapter } from 'rt-platforms'
+import { InteropTopics, Platform, platformHasFeature, PlatformWindow } from 'rt-platforms'
 import { stringify } from 'query-string'
 import { defaultConfig, windowOrigin } from './defaultWindowConfig'
 import { BlotterFilters, validateFilters } from '../../MainRoute/widgets/blotter'
 
-let blotterWindow: Window | null = null
+let openedWindow: PlatformWindow | undefined
```

Comment:
notice that now instead of the old web `Window` type, we are using `PlatformWindow` - this allows us to have more controls of the window that has been opened via platfrom